src/components/DelegatesTable/DelegateVoteCheckbox.vue

```diff
@@ -26,33 +26,53 @@ export default {
       required: true
     }
   },
-  setup(props) {
+  setup (props) {
     const { delegate } = props
     const store = useStore()
 
     const className = 'delegate-vote-checkbox'
-    const classes = {
+    const voteName = 'mdi-thumb'
+    const classes = reactive({
       root: className,
       icon: `${className}__icon`
-    }
+    })
+    const votes = reactive({
+      root: voteName,
+      upVote: `${voteName}-up`,
+      downVote: `${voteName}-up-outline`
+    })
 
     const voted = computed(() => delegate._voted)
+    const originalVoted = voted.value;
```

Comment:
remove `const originalVoted` and use `voted.value` below

Reply:
it doesn't work without this constant, because the `voted.value` changes everytime you click on like button

Reply:
Get it, so you intentionally got rid of reactivity. Then leave as it is 👍